HelpSource/Classes/Phasor.schelp

```diff
@@ -58,6 +58,21 @@ Examples::
 
 code::
 
+// basic: ramp between 'start' and 'end'
+// note that the ramp ends at 9, not 10
+{ Phasor.kr(0, 1, start: 0, end: 10) }.plot(0.03)
+
+// on trigger: ramp jumps to 'resetPos', goes until 'end' and wraps back to 'start'
+// note that the ramp starts from 'resetPos', since it receives a trigger immediately on its first sample
+(
+{
+	var trig = Impulse.kr(ControlRate.ir/5);
+	var ramp = Phasor.kr(trig, 1, start: 0, end: 10, resetPos: 7);
+	[ramp, trig * 10]
+}.plot(0.03)
+)
```

Comment:
or `\dstems` for illustrating timing of triggers that are impulses:
```supercollider
(
{
	var trig = Impulse.kr(ControlRate.ir/8);
	var ramp = Phasor.kr(trig, 1, start: 0, end: 10, resetPos: 7);
	[ramp, trig * 10]
}.plot(0.03).plotMode_([\dlines, \dstems])
)
```

Reply:
Actually, by playing with this it made it more obvious that actually it's reseting to `resetPos + 1`, not `reset` as it should...